llama_stack/apis/inspect/inspect.py

```diff
@@ -20,15 +21,15 @@ class RouteInfo(BaseModel):
 
 @json_schema_type
 class HealthInfo(BaseModel):
-    status: str
-    # TODO: add a provider level status
+    status: HealthStatus
 
 
 @json_schema_type
 class ProviderInfo(BaseModel):
```

Comment:
I did not notice this earlier but now with @cdoern's changes, we have two exactly same class names registered with `ProviderInfo`. I guess one of them is going away with a deprecation soon, so that should be OK I guess.

Reply:
yup, I left both as I didn't want to cause import weirdness, and so that the removal is much cleaner, but I agree its weird to have both for too long